scripts/Compile_LSOA_RUC.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Mar 15 15:05:08 2023
+
+Script that takes to Govt. LSOA --> Urban/Rural classifications and creates an
+area type lookup for the OTP tool to assess an appropriate maximum radial filter 
+distance
+
+@author: Signalis
+
+
+Classification                                  | Classification Code
+Urban major conurbation                         | A1
+Urban minor conurbation                         | B1
+Urban city and town                             | C1
+Urban city and town in a sparse setting         | C2
+Rural town and fringe                           | D1
+Rural town and fringe in a sparse setting       | D2
+Rural village and dispersed                     | E1
+Rural village and dispersed in a sparse setting | E2
+
+In general though:
+    Urban -- [A1, B1, C1, C2]
+    Rural -- [D1, D2, E1, E2]
+
+
+"""
+
+#### IMPORTS ####
+# System imports
+import os 
+
+# 3rd party imports
+import pandas as pd 
+
+#### CONSTANTS ####
+# Specify data directory & filename for Rural Urban Classifications (RUC) file
+DATA_DIR = r"E:\otp4gb-py\Data"
```

Comment:
Use `pathlib.Path` instead of strings for paths, they provide lots of functionality for manipulating and joining paths together